src/components/app.tsx

```diff
@@ -340,7 +340,9 @@ export class AppComponent extends BaseComponent<IProps, IState> {
     const scenarioData = (Scenarios as {[key: string]: Scenario})[scenario];
     const simMargin = { x: 5, y: 5 };
     const simMarginStr = `${simMargin.y}px ${simMargin.x}px 0 ${simMargin.y}px`;
-    const lavaCoderHeight = mapHeight - 2 * simMargin.y;
+    // the widgets panel has been removed from LavaCoder, so the map expands to fill the space
+    const kWidgetsPanelHeight = 160;
+    const lavaCoderHeight = mapHeight + kWidgetsPanelHeight - 2 * simMargin.y;
```

Comment:
This section is becoming a bit of a mess. I think we should try to clean it up:
- Define constants for widget and tab button (is there a better name for this?) height.
- Define `mapHeight` and `tabHeight` in terms of `height` and the above constants.
- `lavaCoderHeight` should then be `tabHeight - 2 * simMargin.y`.